docs/usage/from-a-terminal.rst

```diff
@@ -1057,12 +1062,12 @@ Global overview
                                 Configured value: False
         -t LOOKUP__TIMEOUT, --timeout LOOKUP__TIMEOUT
                                 Sets the default timeout to apply to each lookup
-                                utilities everytime it is possible to define a timeout.
+                                utilities every time it is possible to define a timeout.
                                 Configured value: 5
         -ua USER_AGENT__CUSTOM, --user-agent USER_AGENT__CUSTOM
                                 Sets the user agent to use.
 
-                                If not given, we try to get the lastest (automatically) for you.
+                                If not given, we try to get the latest (automatically) for you.
```

Comment:
Please also apply the change at: 
https://github.com/funilrys/PyFunceble/blob/0d5e5450473aff6712277d8dd0bbc240e2f953f1/PyFunceble/cli/entry_points/pyfunceble/cli.py#L437-L438